ports/espressif/boards/sunton_esp32_2432S032C/mpconfigboard.mk

```diff
@@ -0,0 +1,15 @@
+CIRCUITPY_CREATOR_ID = 0x19991000
+CIRCUITPY_CREATION_ID = 0x00AA032C
+
+IDF_TARGET = esp32
+
+CIRCUITPY_ESP_FLASH_MODE = dio
```

Comment:
Are you sure it is DIO? I think most boards are actually QIO. DIO is only used when loading.

Reply:
I think this is a misunderstanding from my side. I will change this.